charts/surrealdb/templates/service.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.service.port }}
-      targetPort: http
+    - name: http
+      port: {{ .Values.service.port }}
+      targetPort: {{ .Values.service.targetPort }}
```

Comment:
the `targetPort` attribute doesn't exist. Either we add it to the values.yaml file or we default here to `http` if it's not set

Reply:
Well, it defaults to `http` per your last review. I re-added to the `values.yaml`. I suppose it is better when explicit.